build/Build.Pack.cs

```diff
@@ -649,4 +649,58 @@ string GetMicrok8sIpAddress()
         { "linux-arm64", ("arm64", "arm64") },
         { "linux-arm", ("armhf", "armv7") }
     };
+
+    AbsolutePath InstallOctopusCli()
+    {
+        const string cliVersion = "2.11.0";
```

Comment:
Currently using a fixed version of the CLI. This gives some assurances regarding consistency and removes any complexities that arise from having to identify the current version.

It can be updated easily, although we may wish to make this constant more visible.